plugins/googlesignin/class.googlesignin.plugin.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license Proprietary
+ */
+
+
+/**
+ * Class GoogleSignInPlugin
+ *
+ * Expose the functionality of the core class Gdn_OAuth2 to create SSO workflows.
+ */
+
+class GoogleSignInPlugin extends Gdn_OAuth2 {
+    CONST AUTHORIZEURL = 'https://accounts.google.com/o/oauth2/v2/auth';
+    CONST TOKENURL = 'https://oauth2.googleapis.com/token';
+    CONST PROFILEURL = 'https://openidconnect.googleapis.com/v1/userinfo';
+    CONST PROFILENAME = 'name';
+    CONST ACCEPTEDSCOPE = 'email openid profile';
+    /**
```

Comment:
Code sniffer check failed because of  `const` keyword, should be lowercase.
And i would recommend to use regular php naming convention for const names with underscore as a word separator:
```
const AUTHORIZE_URL = 'https://accounts.google.com/o/oauth2/v2/auth';
    const TOKEN_URL = 'https://oauth2.googleapis.com/token';
    const PROFILE_URL = 'https://openidconnect.googleapis.com/v1/userinfo';
    const PROFILE_NAME = 'name';
    const ACCEPTED_SCOPE = 'email openid profile';
```